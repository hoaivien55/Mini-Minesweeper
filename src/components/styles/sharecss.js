import { css } from "styled-components";

const center = css`
	position: absolute;
	left: 50%;
	top: 50%;
	transform: translate(-50%, -50%);
`;

export { center };
