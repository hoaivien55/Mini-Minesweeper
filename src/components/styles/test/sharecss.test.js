import renderer from "react-test-renderer";
import styled from "styled-components";

import { center } from "../sharecss";

const Button = styled.button`
	${center}
`;

it("sharecss: Virtual DOM snapshot", () => {
	const tree = renderer.create(<Button />).toJSON();
	expect(tree).toMatchSnapshot();
});
