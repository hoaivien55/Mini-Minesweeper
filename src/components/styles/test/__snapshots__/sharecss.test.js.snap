// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sharecss: Virtual DOM snapshot 1`] = `
.c0 {
  position: absolute;
  left: 50%;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
}

<button
  className="c0"
/>
`;
